### PR TITLE
Skip loading StaticCatalogStore and AccessControlManager for RM

### PR DESCRIPTION
**Pull request by swapsmagic**
- Skip loading StatigCatalogStore and AccessControlManager for resource manager
- Adding resource manager support in /v1/info endpoint

Test plan - On Test Cluster

```
== NO RELEASE NOTE ==
```

### CODE REVIEW

File: presto-main/src/main/java/com/facebook/presto/server/ServerInfoResource.java [hunk 1]
```diff
@@ -69,7 +71,7 @@ public ServerInfoResource(NodeVersion nodeVersion, NodeInfo nodeInfo, ServerConf
     @Produces(APPLICATION_JSON)
     public ServerInfo getInfo()
     {
-        boolean starting = !catalogStore.areCatalogsLoaded();
+        boolean starting = resourceManager ? true : !catalogStore.areCatalogsLoaded();
```
Comment (tdcmeehan) [inline presto-main/src/main/java/com/facebook/presto/server/ServerInfoResource.java:74]:
```suggestion
        boolean starting = !resourceManager && !catalogStore.areCatalogsLoaded();
```

**Review by tdcmeehan — APPROVED** [review]

File: presto-main/src/main/java/com/facebook/presto/server/ServerInfoResource.java (hunk 1 above)
Comment (tdcmeehan) [reply presto-main/src/main/java/com/facebook/presto/server/ServerInfoResource.java:74]: By the way, why have this be true for RM?  Should it not be false?

Comment (swapsmagic) [reply presto-main/src/main/java/com/facebook/presto/server/ServerInfoResource.java:74]: Not sure how the flag is been used, but if the server is up thought this has to be true. Do you have more context on and if we should keep this flag false or true for RM?

Comment (tdcmeehan) [reply presto-main/src/main/java/com/facebook/presto/server/ServerInfoResource.java:74]: I think `starting` indicates some sort of initialization is occurring.  There's no concept of that in RM right now, so my sense is it should always be `false` until that concept is developed and fully expressed in the code.

**Pull request merged**